### ISSUE TITLE
Update `issues.dlang.org` code links to their GitHub issue counterparts

### ISSUE THREAD

**Issue by thewilsonator** [dlang/dmd#20599]
e.g. https://github.com/dlang/dmd/blob/master/druntime/src/core/internal/traits.d#L270

`$ git grep "issues\.dlang\.org" | wc -l` gives 1206 for phobos and 4311 for DMD.

cc @burner I take it you have some bugzilla scraping code around that could help automate this

**Comment by burner** [comment]
> cc @burner I take it you have some bugzilla scraping code around that could help automate this

I do, you can find it here

https://github.com/burner/D_bugzilla_to_github/blob/master/source/rest.d

**Comment by dkorpel** [comment]
Most links in dmd code / test cases refer to already fixed issues, which haven't been transferred to GitHub. The links still work and now point to a static bugzilla archive. Is there still useful work to do here?

**Comment by ibuclaw** [comment]
No. The old issue links are functional again. The recently added redirects to the archive is better than a bugzilla site that only had 51% uptime.

https://updown.io/cey6

(Now we're back to 100% uptime since 21st March)
